happi/item.py

```diff
@@ -348,6 +349,15 @@ def show_info(self, handle=sys.stdout):
         for key in public_keys:
             pt.add_row([key, show_info[key]])
 
+        # initialize table so we can grab widths
+        _ = pt.get_string()
+
+        # set width of Value column dynamically
+        # set width(Value) = terminal width - width(EntryInfo column)
+        entry_info_width = pt._widths[0] + 10  # account for padding
+        term_width = os.get_terminal_size()[0]
```

Comment:
I seem to recall that `get_terminal_size` can fail if using a program in non-interactive mode (e.g., piping its output to a file). Can you do a quick check of `happi search * > output.txt` for example?

I'd expect you'll need a try/except around this for that case.

Reply:
Excellent catch.  Fix on the way